util/pom.xml

```diff
@@ -52,6 +54,26 @@
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-context</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot</artifactId>
```

Comment:
Do we need all of these in the util module?

Reply:
Yes, if we want to have the shared health indicators here. We could also move them to gateway, which would also move the dependencies.

Reply:
Probably fine to move them into gateway. I assume we don't need them somewhere else?